src/zenml/zen_server/deploy/helm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-name: zenml
-version: "1.0"
+name: zenml-server
+version: "0.42.1"
```

Comment:
SHouldn't this be 0.43.0 already? How do we do it moving forward. DOes this get manually updated as well? If Not, could it make sense to leave the version number empty?

Reply:
yes, it does get updated autonomously as part of the CI. A step fetches the latest verison and updates this file before it is packaged. You're right, it makes sense to leave it empty; I'll make the change.